src/content/en/news/status-update10.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Status Update"
+draft: false
+slug: "status-update"
+date: 2020-10-25
+type: "page"
+---
+
+We have processed a total of 145 submissions so far, with 14 of them being eligible for a payout.
+
+We want to thank Javed Khan (Core Developer at Handshake Development Inc) and Braydon Fuller (Bitcoin Protocol Engineer at PurseIO, Inc) who reported the [INVDoS](https://invdos.net/) vulnerability to the program. HOF has also been updated (primary submitter has been added)
```

Comment:
No need to abbreviate Hall of Fame imo. It took me a moment to work that out, and I am already totally familiar with the bounty program. Finish the sentence with a full stop too :+1:

Reply:
should be fixed now